domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -400,6 +403,10 @@ constructor(
     override suspend fun startVideoRecording(
         onVideoRecord: (CameraUseCase.OnVideoRecordEvent) -> Unit
     ) {
+        if (videoCaptureUseCase == null) {
+            Log.e(TAG, "attempted video recording with null videoCapture use case")
+            return
```

Comment:
This should throw an exception or return an error in the `onVideoRecord` callback. Failing silently could cause issues.